docs/excel/custom-functions-troubleshooting.md

```diff
@@ -33,9 +33,25 @@ Information about custom functions is cached by Office. Sometimes while developi
 
 ## Common issues
 
+### Can't open add-in from localhost: use a local loopback exception
+
+If you see the error "We can't open this add-in from localhost," you will need to enable a local loopback exception. For details on how to do this, see [this Microsoft support article](https://support.microsoft.com/en-us/help/4490419/local-loopback-exemption-does-not-work).
+
+### Runtime logging reports "TypeError: Network request failed" on Excel on Windows
+
+If you see the error "TypeError: Network request failed" in your [runtime log](custom-functions-troubleshooting.md#enable-runtime-logging) while making calls to your localhost server, you'll need to enable a local loopback exception. For details on how to do this, see *Option #2* in [this Microsoft support article](https://support.microsoft.com/en-us/help/4490419/local-loopback-exemption-does-not-work).
+
+### Ensure promises return
+
+When Excel is waiting for a custom function to complete, it displays #BUSY! in the cell. If your custom function code returns a promise, but the promise does not return a result, Excel will continue showing #BUSY!. Check your functions to make sure that any promises are properly returning a result to a cell.
+
+### Error: The dev server is already running on port 3000
+
+Sometimes when running `npm start` you may see an error that the dev server is already running on port 3000 (or whichever port your add-in uses). You can stop the dev server by running `npm stop` or by closing the Node.js window. But in some cases in can take a few minutes for the dev server to actually stop running.
+
 ### My functions won't load: associate functions
 
-In your custom functions' script file, you need to associate each custom function with its ID specified in the [JSON metadata file](custom-functions-json.md). This is done by using the `CustomFunctions.associate()` method. Typically this method call is made after each function or at the end of the script file. If a custom function is not associated, it will not work.
+If you have written your JSON by hand instead of using JSDoc comments to autogenerate your JSON, you need to associate each custom function with its ID specified in the [JSON metadata file](custom-functions-json.md). This is done by using the `CustomFunctions.associate()` method. Typically this method call is made after each function or at the end of the script file. If a custom function is not associated, it will not work.
```

Comment:
If possible it would be best to lead with the actual symptom (isn't it seeing #VALUE?) 

Also I would avoid the phrasing "hand" written or authored. It's a bit awkward and I'm not sure how that would translate. Maybe "if you do not use code comments to generate the JSON metadata" or "if you manage JSON metadata by directly editing the file..."